odl/solvers/nonsmooth/douglas_rachford.py

```diff
@@ -89,19 +91,28 @@ def douglas_rachford_pd(x, f, g, L, tau, sigma, niter,
     can be obtained by setting
 
     .. math::
-        l(x) = 0 \\text{ if } x = 0, \infty \\text{ else.}
+        l(x) = 0 \text{ if } x = 0, \infty \text{ else.}
```

Comment:
Finally, no more annoying double backslashes!

Reply:
We really should make an across the board change for this